README.md

```diff
@@ -172,7 +174,7 @@ Release v1.18.0 contains 5 new collectors, 16 bug fixes, 27 improvements, and 20
 
 The **database engine** is now the default method of storing metrics in Netdata. You immediately get more efficient and configurable long-term metrics storage without any work on your part. By saving recent metrics in RAM and "spilling" historical metrics to disk for long-term storage, the database engine is laying the foundation for many more improvements to distributed metrics.
 
-We even have a [tutorial](https://docs.netdata.cloud/docs/tutorials/longer-metrics-storage/) on switching to the database engine and getting the most from it. Or, just read up on [how performant](https://docs.netdata.cloud/database/engine/#evaluation) the database engine really is.
+We even have a [tutorial](https://docs.netdata.cloud/docs/tutorials/longer-metrics-storage/) on switching to the database engine and getting the most from it. Or, just read up on [how performant](https://docs.netdata.cloud/database/engine/#evaluation) the database engine is.
```

Comment:
Is ending in a preposition deliberate? - it seems ok in the informal case.

Reply:
I'm a pretty informal writer! But I did fix this to make it read a little nicer.